docs/source/integrations/integrations.rst

```diff
@@ -5,5 +5,8 @@ INTEGRATIONS
    :maxdepth: 2
    :glob:
 
-   *
+   bitmovin.md
+   jwplayer.md
+   Theoplayer.md
+   ios-fairplay-sdk.md
```

Comment:
Inconsistent naming, kebab-case

Reply:
needed due to weirdness in read the docs making theoplayer disappear